apps/amethyst/app/(tabs)/settings/index.tsx

```diff
@@ -30,11 +37,31 @@ export default function Settings() {
           selectedValue={colorScheme}
           setSelectedValue={setColorScheme}
         />
+        <TextInputRow
+          labelText="Appview DID"
+          initialValue={appviewDid || ""} // Ensure currentValue is a string
+          onSubmit={(e) => setAppviewDid(e)}
+          placeholder="Enter your Appview DID (e.g., did:plc:...)"
```

Comment:
aren't appviews usually did:web 😜
```suggestion
          placeholder="Enter your Appview DID (e.g., did:web:...)"
```

Reply:
true but technically can be both haha, will change it either way